tests/test_qa_util.py

```diff
@@ -2,22 +2,23 @@
 Contains tests for functions found within qa_util.py
 """
 
-from happytransformer.qa_util import biggest_sums, SumPair
-
+from happytransformer.runners.runner_answer_question import AnswerQuestionRunner
 def test_biggest_sums():
     """
     Tests the biggest_sums function
     """
     items_a = [7, 4, 3]
     items_b = [7, 6, 4]
+    qa_runner = AnswerQuestionRunner("bert", None, None)
+
     expected_pairs = [
-        SumPair(idx1=0, idx2=0, sum=14),  # 7+7
-        SumPair(idx1=0, idx2=1, sum=13),  # 7+6
-        SumPair(idx1=0, idx2=2, sum=11),  # 7+4
-        SumPair(idx1=1, idx2=2, sum=8),  # 4+4
-        SumPair(idx1=2, idx2=2, sum=7)  # 3+4
+        qa_runner.SumPair(idx1=0, idx2=0, sum=14),  # 7+7
+        qa_runner.SumPair(idx1=0, idx2=1, sum=13),  # 7+6
+        qa_runner.SumPair(idx1=0, idx2=2, sum=11),  # 7+4
+        qa_runner.SumPair(idx1=1, idx2=2, sum=8),  # 4+4
+        qa_runner.SumPair(idx1=2, idx2=2, sum=7)  # 3+4
     ]
-    computed_pairs = biggest_sums(items_a, items_b)
+    computed_pairs = qa_runner.biggest_sums(items_a, items_b)
```

Comment:
This method should be a standalone function; it has no inputs other than items_a and items_b

Reply:
Moved to runner_util.py